workloads/files/workload-nodevertical-script-cm.yml

```diff
@@ -81,7 +81,8 @@ data:
       result_dir=${benchmark_results_dir}
     fi
     start_time=$(date +%s)
-    VIPERCONFIG=/tmp/nodevertical.yaml openshift-tests run-test "[Feature:Performance][Serial][Slow] Load cluster should load the cluster [Suite:openshift]" | tee "${result_dir}/clusterloader.txt"
+    git clone https://github.com/cloud-bulldozer/snafu;
+    VIPERCONFIG=/tmp/nodevertical.yaml python snafu/run_snafu.py -t cl scale-ci --cl-output True --dir "${result_dir}" -p openshift-tests | tee "${result_dir}/clusterloader.txt"
```

Comment:
The nodevertical workload run failed with the following error: 
```
Thu Dec 19 19:20:10 UTC 2019 Configuring pbench for NodeVertical
Thu Dec 19 19:20:10 UTC 2019 Done configuring pbench NodeVertical
Thu Dec 19 19:20:10 UTC 2019 Configuring NodeVertical test
Thu Dec 19 19:20:10 UTC 2019 Done configuring NodeVertical test
Thu Dec 19 19:20:10 UTC 2019 Running NodeVertical workload
fatal: could not create work tree dir 'snafu'.: Permission denied
python: can't open file 'snafu/run_snafu.py': [Errno 2] No such file or directory
Thu Dec 19 19:20:10 UTC 2019 Writing Cluster Loader Exit Code
Thu Dec 19 19:20:10 UTC 2019 Writing Cluster Loader Metrics to clusterloader.json
Thu Dec 19 19:20:10 UTC 2019 Finished workload script
Thu Dec 19 19:20:10 UTC 2019 Completed NodeVertical workload run
Thu Dec 19 19:20:10 UTC 2019 Checking Test Results
Thu Dec 19 19:20:10 UTC 2019 Checking Cluster Loader Exit Code
Thu Dec 19 19:20:10 UTC 2019 Cluster Loader Failure
Thu Dec 19 19:20:10 UTC 2019 Test Analysis: Failed
```
Am I missing something?

Reply:
not sure so the issue seems to be with cloning also I opened [PR](https://github.com/openshift-scale/images/pull/27) to sort of address this. Once its' merged we can test again.